lib/routes/routeBackbeat.js

```diff
@@ -120,6 +120,12 @@ function putMetadata(request, response, bucketInfo, objMd, log, callback) {
             // FIXME: add error type MalformedJSON
             return callback(errors.MalformedPOSTRequest);
         }
+        const { headers, bucketName, objectKey } = request;
+        // check if it's metadata only operation
+        if (headers['x-scal-replication-content'] === 'METADATA') {
+            // copy original data locations
+            omVal.location = Array.from(objMd.location);
```

Comment:
Do we need a check if objMd is `undefined`? Looking here:
https://github.com/scality/S3/blob/93d20912a2e58fd389b7ec9765043fa2adb2779a/lib/metadata/metadataUtils.js#L196